src/App.js

```diff
@@ -1,46 +1,39 @@
+/**
+ * This organizes all the UI component of the GUI
+ **the LeftPanel import manages the left side of the GUI (dot languate input and controls)
+ **the RightPanel import manages the right side of the GUI (dot graph display and the implementation details)
+ */
 import React, { useEffect, useState, Component } from "react";
 //this panel holds the graph source (user input)
 import LeftPanel from './component/LeftPanel';
 //this pannel holds the task graph and the implementation details
 import RightPanel from './component/RightPanel';
-import { parseDotLanguageInput } from "./adapter/homeAdapter";
+//this import handles the logic of the GUI
+import { parseDotLanguageInput, defualt_dedup_input } from "./adapter/homeAdapter";
 
 class App extends Component {
 
   constructor(props){  
     //TODO: add a file to show how things work on the high level
     super(props); 
 
-    //set the initial state  
+    //set the initial GUI state 
     this.state = {
       //the pannel input is for saving the user's source of the graph being navigated
-      unprocessed_input: this.defualt_input(),
-      //the state of the graph being navigated --(updated when the user runs any input change)
-      processed_input: "",//this.defualt_input(),

```

Comment:
Any mentioning of "dedup" should not be part of the code at all, b/c dedup is part of the input data. The code should not be input specific.